hw1/01-fizzbuzz.js

```diff
@@ -11,6 +11,39 @@ Use console.log() to write the proper output to the command line.
 
```

Comment:
Hi Samuel,
I think you did a great job with the second method. It's creative and only uses 1 console log command with 1 single variable. 
Personally, I still prefer the 1st method because to me, using the console log statements for each case without managing variables is simpler and more effective. As mentioned above, both ways are good and I love what you've done with the second one.